sound/soc/intel/boards/sof_sdw.c

```diff
@@ -665,9 +665,17 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 		.callback = sof_sdw_quirk_cb,
 		.matches = {
 			DMI_MATCH(DMI_SYS_VENDOR, "LENOVO"),
-			DMI_EXACT_MATCH(DMI_PRODUCT_SKU, "233B")
```

Comment:
@rfvirgil not sure it matter but shouldnt the Fixes tag go before the Sign-off?

Reply:
@ranj063 I couldn't find anything in the kernel documentation to say which order the tags should appear.
The doc:
https://www.kernel.org/doc/html/v6.12/process/submitting-patches.html#using-reported-by-tested-by-reviewed-by-suggested-by-and-fixes

only says that there should be Fixes and Signed-off-by tags, but it doesn't say what order and the example doesn't include a Fixes tag. I've been putting them at the end and none of the maintainers has complained. But I can re-push these patches with the Fixes: first if you prefer.